nio_cli/commands/config.py

```diff
@@ -92,6 +103,28 @@ def config_ssl(name, conf_location):
     os.remove(conf_location)
     os.rename(tmp.name, conf_location)
 
+    success_message(name, True, new_certs.lower() == 'y')
+
+def success_message(name, ssl=False, self_signed=False):
+
+    print('')
+    print('\033[92m' + 'Success!' + '\033[0m')
+    print('First, start your instance by entering your project directory: ' + '\033[94m' + 'cd ' + name + '\033[0m' + ', and starting the nio daemon: '  + '\033[94m' + 'niod' + '\033[0m')
+
+    if (self_signed == True):
+        print('Next, accept your self-signed certificate by visiting ' + '\033[94m' + 'https://localhost:8181' + '\033[0m' + ' and clicking "Advanced > Proceed to Site Anyway".' + '\033[0m')
+
+    if (ssl == True):
+        print('Then, proceed to ' + '\033[94m' + 'https://app.n.io/design' + '\033[0m' + ' and add your local instance to the designer: ')
+        print('- hostname:' + '\033[94m' + ' https://localhost ' + '\033[0m')
```

Comment:
Similar to above, I like the split link for https/http designer, but don't think we need to tell them the host. Docs can do that and it should be self-explanatory. We also can't assume localhost or 8181

Reply:
The cli specifies 'localhost' when it creates the cert... Maybe we let them choose both host and port (localhost:8181 as the default)

Reply:
Yeah, I think if we have them specify host and port to run on that would be cleaner. I didn't realize we were hardcoding localhost for the cert, you're right, we shouldn't be doing that either.

Reply:
that's done in the new push